### PR TITLE
Allow skipping queue creation

### PR DESCRIPTION
**Pull request by mikeminutillo**
Part of #106

### CODE REVIEW

File: src/ServiceControl.Monitoring/Hosting/Commands/SetupCommand.cs [hunk 1]
```diff
@@ -9,6 +9,10 @@ public override Task Execute(Settings settings)
         {
             var endpointConfig = EndpointFactory.PrepareConfiguration(settings);
             endpointConfig.EnableInstallers(settings.Username);
+            if (settings.SkipQueueCreation)
```
Comment (WilliamBZA) [inline src/ServiceControl.Monitoring/Hosting/Commands/SetupCommand.cs]: It's a pity we can't invert the logic easily. I.e. have the default be to not create the queues.

Comment (mikeminutillo) [reply src/ServiceControl.Monitoring/Hosting/Commands/SetupCommand.cs]: @WilliamBZA Why is that an advantage? At the moment we only create queues if you run with the `--setup` flag. It's assumed that part of the setup of the endpoint is to migrate any internal data stores, create folders on disk, create queues, etc.

Comment (WilliamBZA) [reply src/ServiceControl.Monitoring/Hosting/Commands/SetupCommand.cs]: Opt in feels better to me than opt out - especially for things that have a material impact on the environment. No string feelings about it though.

**Comment by mikeminutillo** [comment]
Rebased on develop

**Pull request merged**